cmd/maintenance.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"github.com/urfave/cli/v2"
+
+	"github.com/Scalingo/cli/db/maintenance"
+	"github.com/Scalingo/cli/detect"
+	"github.com/Scalingo/cli/utils"
+	scalingo "github.com/Scalingo/go-scalingo/v6"
+)
+
+var databaseMaintenanceList = cli.Command{
+	Name:     "database-maintenance-list",
+	Category: "Addons Maintenance",
+	Usage:    "List database maintenance",
+	Flags: []cli.Flag{
+		&appFlag,
+		&addonFlag,
+		&cli.IntFlag{Name: "page", Usage: "Page to display", Value: 1},
+		&cli.IntFlag{Name: "per-page", Usage: "Number of deployments to display", Value: 20},
+	},
+	Description: CommandDescription{
+		Description: "List database maintenance",
+		Examples: []string{
+			"scalingo --app my-app --addon addon-id database-maintenance-list",
+			"scalingo --app my-app --addon addon-id database-maintenance-list --per-page 20 --page 5",
```

Comment:
Nitpick: It seems that we usually use `uuid` instead of `id` instead.
```suggestion
			"scalingo --app my-app --addon addon-uuid database-maintenance-list",
			"scalingo --app my-app --addon addon-uuid database-maintenance-list --per-page 20 --page 5",
```